### ISSUE TITLE
[feature] depsyncer add sync all dep

### ISSUE THREAD

**Issue by JieAnthony** [micromdm/nanodep#47]
@jessepeterson 
New option `--all` is added to indicate operating all.
All I can think of is reading the database every time.
Although this may bring additional overhead

**Comment by jessepeterson** [comment]
Yeah -- You could definitely script this. If you're using MySQL just have it read-out the DEP names with `mysql` and restart itself every hour or something. Could also do that for the file backend. But I've noted this enhancement and we can consider the API support to query what we need and possibly implement this. Don't hold your breath, though. :)
